src/Scrollyteller/index.js

```diff
@@ -91,6 +91,14 @@ class Scrollyteller extends React.Component {
 
       this.setState({ sticky });
     }
+
+    // On scroll call a function with pixels and current closest panel
+    if (scrollTween && typeof scrollTween === 'function') {
+      const panel = closestReference.element.getBoundingClientRect();
```

Comment:
You'll have the height of the panel here and should be able to use it to get a normalised value of the 'in view' percentage past the fold.

Reply:
I used a different method because when we scroll down there is no previous panel so we can't get the distance between them. I figured there will 99.999% of the time be at least 2 panels, so I get the distance between those panels (sometimes the distance changes on browser resize). and then calculate the scroll progress between the two using the pixels above the fold, then we can pass the progress to the function in the end.

Seems to work anyway.

Reply:
But yeah, height of panel could work too, but I have noticed that Odyssey handles piecemeal panels differently giving them different top and bottom margins etc it seems, so this may be .

....


OK after our chat... going to work out using height of element to take into account different size paragraph text, notably very long paragraph text which will change the distance between each panel.